### PR TITLE
Gui inbound nightly

### PR DESCRIPTION
**Pull request by parkouss**

### CODE REVIEW

File: mozregression/fetch_configs.py [hunk 1]
```diff
@@ -135,6 +136,25 @@ def set_build_type(self, build_type):
...
+        If not set or set to None, default repos would be used (see
+        :meth:`get_nightly_repo` and :attr:`inbound_branch`)
+        """
+        self.repo = branches.get_name(repo) if repo else None
+
+    def should_use_taskcluster(self):
+        """
+        Returns true if taskcluster should be used as the bisection method.
+
+        Should be only called when the repo and the build type are set.
```
Comment (wlach) [inline mozregression/fetch_configs.py]: Do you want to add an assertion about this?

Comment (wlach) [inline mozregression/fetch_configs.py]: (might help for debugging)

File: mozregression/fetch_configs.py [hunk 2]
```diff
@@ -135,6 +136,25 @@ def set_build_type(self, build_type):
...
+        Allow to define the repo name.
+
+        If not set or set to None, default repos would be used (see
+        :meth:`get_nightly_repo` and :attr:`inbound_branch`)
+        """
+        self.repo = branches.get_name(repo) if repo else None
+
+    def should_use_taskcluster(self):
+        """
+        Returns true if taskcluster should be used as the bisection method.
```
Comment (wlach) [inline mozregression/fetch_configs.py]: true -> True

File: mozregression/fetch_configs.py (hunk 1 above)
Comment (parkouss) [inline mozregression/fetch_configs.py]: Well, in fact it is totally valid to have those not set (e.g. None or 'opt') - this only means that no specific branch or build type has been defined by the user.
I wanted to say that this method relies on those values to return a correct result (if you call it, then change the repo and call it again, the return value may change).
I'll change the comment - thanks for the pointer!

File: mozregression/fetch_configs.py (hunk 2 above)
Comment (parkouss) [inline mozregression/fetch_configs.py]: Will do, thanks :)

**Pull request merged**